tests/functions/module_parser.test.ts

```diff
@@ -0,0 +1,262 @@
+      result
+    );
+    expect(evaluateCell("A1", { A1: `=CONVERT(1, "${from}", "${from}")` })).toBeCloseTo(1);
+    expect(
+      evaluateCell("A1", { A1: `=CONVERT(CONVERT(1, "${from}", "${to}"), "${to}", "${from}")` })
+    ).toBeCloseTo(1);
+  });
+
+  test.each([
+    ["lbm", "kg", "1", 0.453592],
+    ["g", "kg", "1", 0.001],
+    ["ozm", "g", "1", 28.3495],
+    ["stone", "g", "1", 6350.29],
+    ["lbm", "g", "1", 453.592],
+    ["lbm", "ozm", "1", 16],
+    ["ton", "kg", "1", 907.185],
+    ["ton", "lbm", "1", 2000],
+    ["grain", "g", "1", 0.0647989],
+    ["sg", "g", "1", 14593.90294],
+    ["cwt", "g", "1", 45359.237],
+    ["uk_cwt", "g", "1", 50802.3],
+  ])("correctly convert mass unit (%s to %s)", (from, to, value, result) => {
+    expect(evaluateCell("A1", { A1: `=CONVERT(A2, "${from}", "${to}")`, A2: value })).toBeCloseTo(
+      result
+    );
+    expect(evaluateCell("A1", { A1: `=CONVERT(1, "${from}", "${from}")` })).toBeCloseTo(1);
+    expect(
+      evaluateCell("A1", { A1: `=CONVERT(CONVERT(1, "${from}", "${to}"), "${to}", "${from}")` })
+    ).toBeCloseTo(1);
+  });
+
+  test.each([
+    ["F", "C", "68", 20],
+    ["K", "C", "0", -273.15],
+    ["C", "K", "20", 293.15],
+    ["C", "F", "0", 32],
+    ["Rank", "K", "1", 5 / 9],
+    ["Reau", "K", "1", 274.4],
+    ["K", "Reau", "1", -217.72],
+  ])("correctly convert temperature unit (%s to %s)", (from, to, value, result) => {
+    expect(evaluateCell("A1", { A1: `=CONVERT(A2, "${from}", "${to}")`, A2: value })).toBeCloseTo(
+      result
+    );
+    expect(evaluateCell("A1", { A1: `=CONVERT(1, "${from}", "${from}")` })).toBeCloseTo(1);
+    expect(
+      evaluateCell("A1", { A1: `=CONVERT(CONVERT(1, "${from}", "${to}"), "${to}", "${from}")` })
+    ).toBeCloseTo(1);
+  });
+
+  test.each([
+    ["min", "sec", "1", 60],
+    ["hr", "sec", "1", 3600],
+    ["day", "sec", "1", 86400],
+    ["yr", "day", "1", 365.2425],
+  ])("correctly convert time unit (%s to %s)", (from, to, value, result) => {
+    expect(evaluateCell("A1", { A1: `=CONVERT(A2, "${from}", "${to}")`, A2: value })).toBeCloseTo(
+      result
+    );
+    expect(evaluateCell("A1", { A1: `=CONVERT(1, "${from}", "${from}")` })).toBeCloseTo(1);
+    expect(
+      evaluateCell("A1", { A1: `=CONVERT(CONVERT(1, "${from}", "${to}"), "${to}", "${from}")` })
+    ).toBeCloseTo(1);
+  });
+
+  test.each([
+    ["mmHg", "Pa", "1", 133.322],
+    ["Torr", "mmHg", "1", 1],
+    ["psi", "Pa", "1", 6894.76],
+    ["atm", "Pa", "1", 101325],
+  ])("correctly convert pressure unit (%s to %s)", (from, to, value, result) => {
+    expect(evaluateCell("A1", { A1: `=CONVERT(A2, "${from}", "${to}")`, A2: value })).toBeCloseTo(
+      result
+    );
+    expect(evaluateCell("A1", { A1: `=CONVERT(1, "${from}", "${from}")` })).toBeCloseTo(1);
+    expect(
+      evaluateCell("A1", { A1: `=CONVERT(CONVERT(1, "${from}", "${to}"), "${to}", "${from}")` })
+    ).toBeCloseTo(1);
```

Comment:
```suggestion
    expect(evaluateCell("A1", { A1: `=CONVERT(1, "${from}", "${from}")` })).toBeCloseTo(1);
    expect(
      evaluateCell("A2", { A2: `=CONVERT(A1, "${to}", "${from}")` })
    ).toBeCloseTo(1);
```
nitpick, a bit shorter

Reply:
Nope